src/app/admin/page.tsx

```diff
@@ -39,14 +42,30 @@ const getUsers = async (): Promise<UserRow[]> => {
     });
 };
 
+const getCollectionCentres = async (): Promise<Schema["collection_centres"][]> => {
+    const { data, error } = await supabase.from("collection_centres").select();
+
+    // TODO VFB-23 Move error handling of this request to client side
```

Comment:
rebase main and talk to Dan/Daniel about error handling :)

Reply:
Done